src/pages/EditMonitor.vue

```diff
@@ -1133,6 +1188,14 @@ export default {
         return {
             minInterval: MIN_INTERVAL_SECOND,
             maxInterval: MAX_INTERVAL_SECOND,
+            ping_count_min: PING_COUNT_MIN,
+            ping_count_max: PING_COUNT_MAX,
+            packetSize_min: PING_PACKET_SIZE_MIN,
+            packetSize_max: PING_PACKET_SIZE_MAX,
+            ping_deadline_min: PING_DEADLINE_MIN,
+            ping_deadline_max: PING_DEADLINE_MAX,
+            ping_timeout_min: PING_TIMEOUT_MIN,
+            ping_timeout_max: PING_TIMEOUT_MAX,
```

Comment:
I would like for this not being a `data`.

Most other cases where this is happeinng in this project (except for interval) choose to inline these values and duplicate them in this circumstance between front and backend.
Currently the deduplication makes is a bit hard to read what all checks are.

=> lets inline these values as well here. This has the benefit of also making this simper to review